ndless-sdk/toolchain/build_toolchain.sh

```diff
@@ -39,38 +39,42 @@ mkdir -p "$PREFIX"
 # Section 0: prerequisites
 echo 'int main() {return 0;}' > test.c
 error=0
-if ! gcc -lgmp test.c -o test; then error=1; echo 'GMP (gmp-devel/libgmp-dev) dependency seems to be missing!'; fi
+if ! gcc -lgmp  test.c -o test; then error=1; echo 'GMP (gmp-devel/libgmp-dev) dependency seems to be missing!'; fi
 if ! gcc -lmpfr test.c -o test; then error=1; echo 'MPFR (mpfr-devel/libmpfr-dev) dependency seems to be missing!'; fi
-if ! gcc -lmpc test.c -o test; then error=1; echo 'MPC (mpc-devel/libmpc-dev) dependency seems to be missing!'; fi
-if ! gcc -lz test.c -o test; then error=1; echo 'zlib (zlib-devel/zlib1g-dev) dependency seems to be missing!'; fi
+if ! gcc -lmpc  test.c -o test; then error=1; echo 'MPC (mpc-devel/libmpc-dev) dependency seems to be missing!'; fi
+if ! gcc -lz    test.c -o test; then error=1; echo 'zlib (zlib-devel/zlib1g-dev) dependency seems to be missing!'; fi
 #TODO: This test fails on openSUSE
 #if ! gcc -ltermcap test.c -o test; then error=1; echo 'termcap (termcap/libtinfo-dev) dependency seems to be missing!'; fi
 if ! gcc -lpython2.7 test.c -o test; then error=1; echo 'libpython2.7 (python-devel/python2.7-dev) dependency seems to be missing!'; fi
 rm -f test test.c
-if ! makeinfo -h > /dev/null; then error=1; echo 'makeinfo (texinfo) dependency seems to be missing!'; fi
+if ! makeinfo -h > /dev/null;   then error=1; echo 'makeinfo (texinfo) dependency seems to be missing!'; fi
 [ $error -eq 1 ] && exit 1
 
 mkdir -p build download
 
 if [ ! -f .downloaded ]; then
 	wget -c http://ftp.gnu.org/gnu/binutils/$BINUTILS.tar.bz2 -O download/$BINUTILS.tar.bz2 && tar xvjf download/$BINUTILS.tar.bz2 && \
-	wget -c ftp://ftp.gnu.org/gnu/gcc/$GCC/$GCC.tar.bz2 -O download/$GCC.tar.bz2 && tar xvjf download/$GCC.tar.bz2 && \
-	wget -c ftp://ftp.gnu.org/gnu/gdb/$GDB.tar.xz -O download/$GDB.tar.xz && tar xvJf download/$GDB.tar.xz && \
-	wget -c ftp://sourceware.org/pub/newlib/$NEWLIB.tar.gz -O download/$NEWLIB.tar.gz && tar xvzf download/$NEWLIB.tar.gz && \
+	wget -c ftp://ftp.gnu.org/gnu/gcc/$GCC/$GCC.tar.bz2       -O download/$GCC.tar.bz2      && tar xvjf download/$GCC.tar.bz2 && \
+	wget -c ftp://ftp.gnu.org/gnu/gdb/$GDB.tar.xz             -O download/$GDB.tar.xz       && tar xvJf download/$GDB.tar.xz && \
+	wget -c ftp://sourceware.org/pub/newlib/$NEWLIB.tar.gz    -O download/$NEWLIB.tar.gz    && tar xvzf download/$NEWLIB.tar.gz && \
 	touch .downloaded
 	if [ $? -ne 0 ]; then
 		echo "Download failed!"
 		exit 1
 	fi
 fi
 
```

Comment:
Please always initialize variables in bash. Everything should still run fine with ```set +u```